website/content/docs/blueprint-reference/components.md

```diff
@@ -95,7 +95,7 @@ spec:
         kind: manifest
         enabled: true
         manifest:
-          url: "https://raw.githubusercontent.com/kubernetes/website/main/content/en/examples/admin/namespace-dev.yaml"
+          url: "https://raw.githubusercontent.com/metallb/metallb/v0.13.10/config/manifests/metallb-native.yaml"
```

Comment:
Should we update to the latest while we are at it?

https://raw.githubusercontent.com/metallb/metallb/v0.14.3/config/manifests/metallb-native.yaml

Reply:
Actually there might be a URL that always points to the latest so it stays up to date

Reply:
Let me check

Reply:
@nwneisen looks like metallb uses the versioned manifests. Let me update it to the latest version.

Reply:
@nwneisen Done